boards/stm32f4discovery/Makefile.include

```diff
@@ -3,7 +3,7 @@ export CPU = stm32f4
 export CPU_MODEL = stm32f407vg
 
 # set default port depending on operating system
-PORT_LINUX ?= /dev/ttyUSB0
```

Comment:
I know is not related to the PR. I could open a different one if you prefer :-) 

Rationale:
This board appears as `ttyACM0` not as `ttyUSB0` in my machine the same as most Nucleo boards I've tried. I'm not sure this is kernel or distribution dependent (tried on Ubuntu [kernel 3.13] and arch [kernel 4.4]) or were remaining of board copy-paste.